routes/auth.js

```diff
@@ -0,0 +1,12 @@
+const express = require ('express');
+const router = express.Router();
+const authorization = require('../middleware/authorization');
```

Comment:
```suggestion
const {allowTo} = require('../middleware/authorization');
```

Then on the controller, you only have to used it without authorisation.allowTo